code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -163,4 +163,8 @@
 
 /mob/living/carbon/human/get_policy_keywords()
 	. = ..()
-	. += "[dna.species.type]"
\ No newline at end of file
+	. += "[dna.species.type]"
+
+/mob/living/carbon/human/can_see_reagents()
+	. = ..()
+	return . || (isclothing(head) ? head.clothing_flags & SCAN_REAGENTS : FALSE) || (isclothing(glasses) ? glasses.clothing_flags & SCAN_REAGENTS : FALSE)

```

Comment:
Looks like this removes the ability for the mask to scan for reagents? Not that it was used, but it seems fine to have.

I like the bitflag change but this could benefit from being on more than one line.